src/authorizations/components/redesigned/EditTokenOverlay.tsx

```diff
@@ -71,15 +72,20 @@ const EditTokenOverlay: FC<Props> = props => {
     }
   }
 
-  const onSave = () => {
+  const onSave = async () => {
     const {auth, updateAuthorization} = props
 
-    updateAuthorization({
-      ...auth,
-      description: description,
-      status: togglestatus ? 'active' : 'inactive',
-    })
-    handleDismiss()
+    try {
+      await updateAuthorization({
+        ...auth,
+        description: description,
+        status: togglestatus ? 'active' : 'inactive',
+      })
+      event('token.edit.success', {id: auth.id, name: description})
```

Comment:
Could this event be handled in the thunk instead of here? That way we can get rid of the try/catch